gvm/protocols/gmpv7/gmpv7.py

```diff
@@ -3813,10 +3837,13 @@ def get_report_format(self, report_format_id: str) -> Any:
 
         if not report_format_id:
             raise RequiredArgument(
-                function=self.get_report_format.__name__,
+                function=self.verify_report_format.__name__,
```

Comment:
Same here. I think this change is wrong.
```suggestion
                function=self.get_report_format.__name__,
```